charms/worker/k8s/src/charm.py

```diff
@@ -282,8 +283,8 @@ def _bootstrap_k8s_snap(self):
         bootstrap_config = BootstrapConfig()
         self._configure_datastore(bootstrap_config)
         self._configure_cloud_provider(bootstrap_config)
-        bootstrap_config.service_cidr = self.config["service-cidr"]
-        bootstrap_config.control_plane_taints = self.config["register-with-taints"].split()
+        bootstrap_config.service_cidr = str(self.config["service-cidr"])
+        bootstrap_config.control_plane_taints = str(self.config["register-with-taints"]).split()
```

Comment:
ops.model.config now returns a type called `ConfigData` that models config data.  There's no clear policy on how to get to that type info -- so we'll just be explicit here that these config values are strings